src/Symfony/Bundle/TwigBundle/Loader/FilesystemLoader.php

```diff
@@ -89,7 +90,7 @@ protected function findTemplate($template, $throw = true)
         }
 
         if (false === $file || null === $file) {
-            throw new \Twig_Error_Loader(sprintf('Unable to find template "%s".', $logicalName), -1, null, $previous);
+            throw new \Twig_Error_Loader($errorMessage, -1, null, $previous);
```

Comment:
The problem is that there are two `$previous`. The first one is "the good one" ... but we always override it. If I use `throw $previous` the error message is worse:

![worse_error_message](https://cloud.githubusercontent.com/assets/73419/12621346/7627a7ca-c520-11e5-8e51-223e1c1015bb.png)